source/js/buyers-guide/components/creepometer/creepometer.jsx

```diff
@@ -73,7 +73,18 @@ export default class Creepometer extends React.Component {
     });
   }
 
-  slideClick(e) {
+  slideFromKey(e) {
+    const k = e.key;
+    let p = this.state.percentage;
+    if (k === `ArrowLeft`) {
+      this.saveTrackHead(p - 5);
+    }
+    if (k === `ArrowRight`) {
+      this.saveTrackHead(p + 5);
+    }
+  }
+
+  slideFromClick(e) {
```

Comment:
@mmmavis this is new code that slides the creepiness on cursor keys. Thank fully, it's pretty simple =)

Reply:
Nice! Works pretty well too!